install_config/topics/configuring_for_aws_registry-proc.adoc

```diff
@@ -80,12 +80,18 @@ openshift_hosted_registry_pullthrough=true
 openshift_hosted_registry_acceptschema2=true
 openshift_hosted_registry_enforcequota=true
 openshift_hosted_registry_replicas=3
+# AWS Registry Encryption Configuration
+openshift_hosted_registry_storage_s3_encrypt
```

Comment:
openshift_hosted_registry_storage_s3_encrypt=false (if we want to have defaults there)

And I'd put it after `openshift_hosted_registry_storage_s3_rootdirectory`, because it's also about s3 storage.

Reply:
@dmage I think we do want defaults, thanks. I added `=false` and moved as indicated.